Blazor_Foundation_6_Test_Shared/Documentation/Containers/Accordion.razor

```diff
@@ -1,6 +1,89 @@
 ﻿@page "/accordion";
 
-<h3>Accordion</h3>
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+@using OpenCodeDev.Blazor.Foundation.Components.Controls;
+@using OpenCodeDev.Blazor.Foundation.Components.Containers;
+@using Microsoft.AspNetCore.Components.Forms;
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Accordion</h2>
+        <p>Accordions are elements that help you organize and navigate multiple documents in a single container. They can be used for switching between items in the container.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>The container for an accordion needs the tag <Code>Accordion</Code>.</p>
+        <p>Inside the <Code>Accordion</Code>, place a series of <Code>AccordionItem</Code> tag. To mark which pane should be open by default, add the attribute <Code>Active</Code> and set it to true</p>
```

Comment:
place a series of <Code>AccordionItem</Code> tag. => place a series of <Code>AccordionItem</Code> component.